Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
+    }
+    environment {
+        // What variables do we need here
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Build') {
+            // Parallel Stage Syntax may soon be changing for declaritive Jenkinsfile(s)
+            // https://issues.jenkins-ci.org/browse/JENKINS-41334
+            steps{
+                parallel (
+                    "linuxBuild" : {
+                        steps {
+                            slackSend color: 'good', message: "STARTED: Job ${BUILD_NUMBER}"
+                            sh """echo Full build of the "$projectName" "$GIT_BRANCH" branch"""
+                            withMaven(maven: 'M3', mavenSettingsConfig: 'codice-maven-settings') {
+                                //TODO update mavenSettingsConfig with a valid config
+                                //sh 'mvn clean install -T 16 -P!documentation -P!findbugs -DskipTests=true -Dpmd.skip=true -Djacoco.skip=true -Dfindbugs.skip=true -Dcheckstyle.skip=true'
+
+                                //run full build here until we can clean the maven phases further
+                                sh 'mvn clean install'
```

Comment:
I think initially all we want to skip itest here using something like ` -pl \!:test-itests-ddf` and then only build/run the itests in the Integration Test stage.